eml.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file (`eml.xml`) and `dwc_occurrence.csv` in the root should not have been committed.

Reply:
You can also remove the files in https://github.com/inbo/movepub/tree/main/inst/sql